datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -376,7 +376,8 @@ public void run() {
           _kafkaTopicPartitionTracker.onPartitionsPolled(records);
 
           Instant readTime = Instant.now();
-          processRecords(records, readTime);
+          long readTimeInNanos = System.nanoTime();
+          processRecords(records, readTime, readTimeInNanos);
```

Comment:
Nitpick. Can't we just pass System.nanoTime() to processRecords directly here cause I dont see we logging this variable in any other way.

Reply:
done